### PR TITLE
fix for 2 out of 4 outdated tests

### PR DESCRIPTION
**Pull request by qaga**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* CRITICAL 2 critical

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* CRITICAL [Use a logger to log this exception.](https://github.com/Wikia/selenium-tests/blob/bc6387bbb8331d1ed0f5bba0c5b9f2f50b210290/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostDetailsPage.java#L96) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)
* CRITICAL [Use a logger to log this exception.](https://github.com/Wikia/selenium-tests/blob/bc6387bbb8331d1ed0f5bba0c5b9f2f50b210290/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostsListPage.java#L189) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

**Comment by justnpT** [comment]
fix is for
anonUserOnDesktopCanNotVoteForPostDetails
anonUserOnMobileCanNotVoteForPostDetails

**Comment by justnpT** [comment]
http://jenkins:8080/job/test-stability-check/56
http://jenkins:8080/job/test-stability-check/57

**Comment by justnpT** [comment]
This branch should be merged with master and the tests that are supposed to get fixed should be reenabled and related issue should be removed

**Comment by tomasznapieralski** [comment]
Remember to enable test you fixed and to remove related issue from them.
@justnpT you were testing stability on disabled tests :)

File: src/test/java/com/wikia/webdriver/testcases/discussions/Upvoting.java [hunk 1]
```diff
@@ -23,8 +23,7 @@
    * ANONS ON MOBILE SECTION
    */
 
-  @Test(groups = "discussions-anonUserOnMobileCanNotVoteForPostDetails", enabled = false)
-  @RelatedIssue(issueID = "SOC-2596")
+  @Test(groups = "discussions-anonUserOnMobileCanNotVoteForPostDetails", enabled = true)
```
Comment (bognix) [inline src/test/java/com/wikia/webdriver/testcases/discussions/Upvoting.java]: `enabled=true` is redundant

File: src/test/java/com/wikia/webdriver/testcases/discussions/Upvoting.java [hunk 2]
```diff
@@ -49,8 +48,7 @@ public void anonUserOnMobileCanNotUpvote() {
    * ANONS ON DESKTOP SECTION
    */
 
-  @Test(groups = "discussions-anonUserOnDesktopCanNotVoteForPostDetails", enabled = false)
-  @RelatedIssue(issueID = "SOC-2596")
+  @Test(groups = "discussions-anonUserOnDesktopCanNotVoteForPostDetails", enabled = true)
```
Comment (bognix) [inline src/test/java/com/wikia/webdriver/testcases/discussions/Upvoting.java]: same as comment above

File: src/test/java/com/wikia/webdriver/testcases/discussions/Upvoting.java [hunk 3]
```diff
@@ -49,8 +48,7 @@ public void anonUserOnMobileCanNotUpvote() {
...
   @Execute(asUser = User.ANONYMOUS)
   @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
```
Comment (bognix) [inline src/test/java/com/wikia/webdriver/testcases/discussions/Upvoting.java]: just out of curiosity, why firefox not chrome?

Comment (qaga) [inline src/test/java/com/wikia/webdriver/testcases/discussions/Upvoting.java]: browserSize does not work on Chrome, red more here :)
https://wikia-inc.atlassian.net/wiki/pages/viewpage.action?pageId=71303284

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* CRITICAL 2 critical

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* CRITICAL [Use a logger to log this exception.](https://github.com/Wikia/selenium-tests/blob/abc304fbed92e2b02c58b8a94692d3abcb0858e2/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostDetailsPage.java#L96) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)
* CRITICAL [Use a logger to log this exception.](https://github.com/Wikia/selenium-tests/blob/abc304fbed92e2b02c58b8a94692d3abcb0858e2/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostsListPage.java#L189) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

**Comment by justnpT** [comment]
@tomasznapieralski I know, here is the rerun:
http://jenkins:8080/job/test-stability-check/58/
http://jenkins:8080/job/test-stability-check/59

**Comment by tomasznapieralski** [comment]
I don't know why sonar is failing here, i've created PR with the same changes and it is passing #1245

**Comment by justnpT** [comment]
We cant merge it yet, this stability check didn't pass:
http://jenkins:8080/job/test-stability-check/61/

**Comment by tomasznapieralski** [comment]
well it was passing twice for me, and i see for you it failed 1/10 and that failure was not caused by test:
http://jenkins:8080/job/test-stability-check/61/count=8,groups=$Group,label=webdriver/artifact/logs/log.html

**Pull request closed without merging**